.drone.star

```diff
@@ -143,8 +144,8 @@ config = {
         "skip": False,
     },
     "rocketchat": {
-        "channel": "ocis-internal",
-        "from_secret": "rocketchat_chat_webhook",
+        "channel": "infinitescale",
+        "from_secret": "rocketchat_talk_webhook",
```

Comment:
we have `rocketchat_talk_webhook` in the organization level

but I got an error after switching: https://drone.owncloud.com/owncloud/ocis/32529/56/1

it's strange because sending to https://talk.owncloud.com/channel/server using `rocketchat_talk_webhook` works correct 
@xoxys @saw-jan maybe you know what is issue?

Reply:
> Maybe the value of `rocketchat_talk_webhook` is not a valid one?

working in core repo. the problem is that the secret is not available for PRs

Reply:
@ScharfViktor you can make the PR ready to review and merge. The changes are only the channel name and webhook which should work